goal_src/engine/camera/cam-start.gc

```diff
@@ -5,3 +5,78 @@
 ;; name in dgo: cam-start
 ;; dgos: GAME, ENGINE
 
+;; definition for function cam-stop
+(defun cam-stop ()
+  (kill-by-name 'camera-master *active-pool*)
+  (kill-by-name 'camera-slave *active-pool*)
+  (kill-by-name 'camera-combiner *active-pool*)
+  (set! *camera* #f)
+  (set! *camera-combiner* #f)
+  (let ((v0-3 cam-string))
+   (set! *camera-base-mode* v0-3)
+   v0-3
+   )
+  )
+
+;; definition for function cam-start
+;; INFO: Return type mismatch int vs none.
+(defun cam-start ((arg0 symbol))
+  (cam-stop)
+  (let ((s5-0 (get-process *camera-dead-pool* camera-combiner #x4000)))
+   (when s5-0
+    (let ((t9-2 (method-of-type camera-combiner activate)))
+     (t9-2
+      (the-as camera-combiner s5-0)
+      *camera-pool*
+      'camera-combiner
+      (the-as pointer #x70004000)
+      )
+     )
+    (let ((t9-3 (the-as (function process function) run-function-in-process))
+          (a0-3 s5-0)
+          )
+     (let ((a1-2 cam-combiner-init))
+      )
+     (t9-3 a0-3)
```

Comment:
In this case, you should case `t9-3` to `(function process object function)`.  

This code;
```
     (let ((a1-2 cam-combiner-init))
      )
 ```

is the second argument to the function.